bears/css/CSSCombBear.py

```diff
@@ -0,0 +1,246 @@
+
+    LANGUAGES = {"CSS"}
+    REQUIREMENTS = {NpmRequirement('csscomb')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/bxpke7nizyxdf6mlx6ss4h405'
+    CAN_FIX = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         remove_empty_rulesets: bool = True,
+                         force_semicolon: bool = True,
+                         upper_case_color: bool = True,
+                         block_indent: str = "\t",
+                         allow_shorthand_color: bool = False,
+                         eof_newline: bool = True,
+                         leading_zero: bool = True,
+                         use_single_quotes: bool = True,
+                         sort_order_fallback: bool = True,
+                         space_before_colon: bool = True,
+                         space_after_colon: bool = True,
+                         space_between_declaration: str = "\n",
+                         space_before_o_brace: bool = True,
+                         space_after_o_brace: bool = True,
+                         space_after_s_delim: str = "\n",
+                         space_before_c_brace: str = "\n",
+                         strip_spaces: bool = True,
+                         use_spaces: bool = False,
+                         vendor_prefix_align: bool = True):
+        """
+        :param filename
+            The filename user wants to use.
+        :param remove_empty_rulesets:
+            Remove empty rulesets if they exist.
+
+            Example:
+
+            ::
+                .a {
+                    color: tomato;
+                }
+                .b{}
+
+            After using the bear:
+
+            ::
+                .a {
+                    color: tomato;
+                }
+        :param force_semicolon:
+            If ``True``, adds a semicolon at the end.
+
+            Example:
+
+            ::
+                .a {
+                    color: tomato
+                }
+
+            After using the bear:
+
+            ::
+                .a {
+                    color: tomato;
+                }
+        :param upper_case_color:
+            If ``True``, make characters for the color upper.
+
+            Example(for upper case):
+
+            ::
+                .a {
+                    color: #fff;
+                }
+
+            After using the bear:
+
+            ::
+                .a {
+                    color: #FFF;
+                }
+        :param block_indent:
+            If set to "\t", indents by a tab.
+            If set to " ", indents by a space.
+
+            Example:
+
+            ::
+                .a {
+                    color: tomato;
+                }
+
+            After using the bear:
+
+            ::
+                .a {
+                    color: tomato;
+                }
```

Comment:
none of your examples have tabs, I think it'd be better to just remove the example here.